### ISSUE TITLE
driver failed programming external connectivity on endpoint mysql

### ISSUE THREAD

**Issue by hecatae** [OpenSIPS/opensips-softswitch-ce#4]
docker compose up
[+] Running 5/5
 ✔ Container opensips     Recreated                                                                                                                     0.2s
 ✔ Container rtpproxy     Recreated                                                                                                                     0.1s
 ✔ Container freeswitch   Recreated                                                                                                                     4.5s
 ✔ Container opensips-cp  Recreated                                                                                                                     0.2s
 ✔ Container mysql        Recreated                                                                                                                     0.1s
Attaching to freeswitch, mysql, opensips, opensips-cp, rtpproxy
Error response from daemon: driver failed programming external connectivity on endpoint mysql (75be3c47a0283cf292d2b9ed72b6f97cacafcabbc4e9a05f10dbbd36f67d4db7): failed to bind port 192.168.1.129:3306/tcp: Error starting userland proxy: listen tcp4 192.168.1.129:3306: bind: cannot assign requested address

nothing in my .env has this ip address, where is it coming from:
NETWORK=172.72.0.0/24

OPENSIPS_IP=172.72.0.2

MYSQL_IP=172.72.0.3
MYSQL_ROOT_PASSWORD=root
MYSQL_DATABASE=opensips
MYSQL_USER=opensips
MYSQL_PASSWORD=opensipsrw
MYSQL_PORT=3306

CP_IP=172.72.0.4

RTPPROXY_IP=172.72.0.5
RTPPROXY_PORT=7899
MINPORT=30000
MAXPORT=30100

FREESWITCH_IP=172.72.0.6

ENABLE_HOMER=no
HOMER_IP=${HOST_IP}
HOMER_PORT=9060

DEFAULT_DOMAIN=domain.sip

**Comment by razvancrainea** [comment]
@hecatae make sure you tune the [HOST_IP](https://github.com/OpenSIPS/opensips-softswitch-ce/blob/main/docs/setup.md#network) variable accordingly.

**Comment by hecatae** [comment]
> @hecatae make sure you tune the [HOST_IP](https://github.com/OpenSIPS/opensips-softswitch-ce/blob/main/docs/setup.md#network) variable accordingly.

ip route
default via 87.106.199.1 dev eth0
87.106.199.1 dev eth0 scope link
172.17.0.0/16 dev docker0 proto kernel scope link src 172.17.0.1 linkdown
172.72.0.0/24 dev br-4fea96435b1e proto kernel scope link src 172.72.0.1 linkdown

cat /etc/hosts
127.0.0.1 localhost
127.0.1.1 debian
::1 localhost ip6-localhost ip6-loopback
ff02::1 ip6-allnodes
ff02::2 ip6-allrouters

Networking is set to DHCP, this is a VPS on the internet, on the public ip subnet 87.106.199.0/24

 ip addr
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group default qlen 1000
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
    inet 127.0.0.1/8 scope host lo
       valid_lft forever preferred_lft forever
    inet6 ::1/128 scope host noprefixroute
       valid_lft forever preferred_lft forever
2: eth0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc fq_codel state UP group default qlen 1000
    link/ether  brd ff:ff:ff:ff:ff:ff
    inet 87.106.199.x/32 brd 87.106.199.x scope global dynamic eth0
       valid_lft 512sec preferred_lft 512sec
    
There's nothing on the host with a 192.168.1 address.

**Comment by razvancrainea** [comment]
Yes, exactly, that's why you need to change the `HOST_IP` in the `.env` file to `87.106.199.x`

**Comment by hecatae** [comment]
> Yes, exactly, that's why you need to change the `HOST_IP` in the `.env` file to `87.106.199.x`

Which is what I have done, yet it still binds to this 192.168.1.129 address, it ignores me changing the HOST_IP line to my VPS public ip.

I've deleted the containers, I've removed the repo sync, I've git cloned again, changed the HOST_IP and ran docker compose up clean, and it still has the address that I've removed from the .env file.

Bit lost here

**Comment by razvancrainea** [comment]
you need to remove the containers: run `docker compose down` then `docker compose up` again.
